### ISSUE TITLE
Emoji support with a custom eslint preset

### ISSUE THREAD

**Issue by jy95** [semantic-release/commit-analyzer#45]
Hello,

I use the conventional-changelog preset eslint with [sgc ](https://github.com/JPeer264/node-semantic-git-commit-cli) and I would to handle a custom thing introduced by sgc : emoji at the beginning of the commit title : How can I do that ?

Now, a build where I use [that ](https://travis-ci.org/jy95/torrent-files-library/jobs/320472340): 

```
[Semantic release]: Call plugin analyze-commits
[Semantic release]: Analyzing commit: :hammer: Refactor: Update description inside package.json
[Semantic release]: The commit should not trigger a release
``` 

and a build without [that ](https://travis-ci.org/jy95/torrent-files-library/jobs/320475836): 

```
[Semantic release]: Call plugin analyze-commits
[Semantic release]: Analyzing commit: Refactor: Remove emoji for sgcrc
[Semantic release]: The release type for the commit is patch 
``` 

My config : 
package.json (and the [rules ](https://github.com/jy95/torrent-files-library/blob/master/config/release-rules.js))
```json
"release": {
    "analyzeCommits": {
      "preset": "eslint",
      "releaseRules": "./config/release-rules.js"
    },
    "generateNotes": {
      "preset": "eslint"
    }
  }
```

**Comment by jy95** [comment]
Found it - the headerPattern option should be edited for that
